Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -2729,7 +2729,10 @@ void CClientGame::AddBuiltInEvents()
     m_Events.AddEvent("onClientMarkerHit", "entity, matchingDimension", NULL, false);
     m_Events.AddEvent("onClientMarkerLeave", "entity, matchingDimension", NULL, false);
 
-    // Marker events
+    m_Events.AddEvent("onClientPlayerMarkerHit", "marker, matchingDimension", NULL, false);
+    m_Events.AddEvent("onClientPlayerMarkerLeave", "marker, matchingDimension", NULL, false);
```

Comment:
nullptr instead of NULL

Reply:
still `nullptr` instead of `NULL`

Reply:
Thanks, I forgot to save it in the previous commit